src/ServiceContainer/SymfonyExtension.php

```diff
@@ -142,8 +163,6 @@ private function loadKernel(ContainerBuilder $container, array $config): void
         $definition->setFile($this->getKernelFile($container->getParameter('paths.base'), $config['path']));
 
         $container->setDefinition(self::KERNEL_ID, $definition);
-
-        $this->requireKernelBootstrapFile($container->getParameter('paths.base'), $config['bootstrap']);
```

Comment:
I guess we can leave it here (as it does nothing if `bootstrap` is null).

Reply:
No, it is done [here](https://github.com/FriendsOfBehat/SymfonyExtension/pull/24/files#diff-63d16ef3a0dae05d4f5902651034de62)